mne/realtime/epochs.py

```diff
@@ -205,9 +215,82 @@ def __init__(self, client, event_id, tmin, tmax, stim_channel='STI 014',
 
     @property
     def events(self):
-        """The events associated with the epochs currently in the queue."""
+        """
+        Returns the events associated with the epochs currently in the queue.
```

Comment:
This is a `property` (not exposed to the user as a function) so I think the original docstring is cleaner